README.md

```diff
@@ -1,2 +1,33 @@
 # Apple-website-webpage-clone
 This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+
+[screenshot](screenshot.PNG)
```

Comment:
- [OPTIONAL] You can directly implement the screenshot of your project to your README instead of adding the screenshot's link.